ovos_PHAL_plugin_oauth/__init__.py

```diff
@@ -58,6 +59,10 @@ def oauth_callback(munged_id):
         ).json()
 
     with OAuthTokenDatabase() as db:
+        if 'expires_at' in token_response:
```

Comment:
Looks like this should be `if 'expires_in' in token_response`?

Reply:
I'm puzzling about this. Google is sending the key `expires_at'

Reply:
just checked googles token fetch response has it both (1 hour token)
```
{'access_token': 'xxx', 'expires_in': 3599, 'scope': ['https://www.googleapis.com/auth/calendar'], 'token_type': 'Bearer', 'expires_at': 1704829227.0002847}
```

Reply:
After checking a bit I believe the spotify response only has the `expires_in`. So it should probably be changed to be

```python
 if 'expires_at' not in token_response:
    token_response['expires_at'] = time.time() + token_response['expires_in']
```

Reply:
this also has to/should be reflected
```diff
+ 224            new_token_data['expires_at'] = time.time() + token_data['expires_in']
```